### PR TITLE
Added french Pizza Paï issue 3679

### PR DESCRIPTION
**Pull request by michalisKout**

### CODE REVIEW

**Comment by Adamant36** [comment]
Hi there and thanks for the contribution. @pyrog are these both fast food and restaurants or just one of them? Going by the pictures of their places in Google search it looks like they are both.

**Comment by michalisKout** [comment]
Hello @Adamant36, your welcome. If a change is necessary for this commit please ping me.

**Comment by pyrog** [comment]
it seem that both **ARE** fast food and restaurants

File: brands/amenity/restaurant.json [hunk 1]
```diff
@@ -2706,6 +2706,18 @@
       "name": "Zizzi"
     }
   },
+  "amenity/restaurant|Pizza Paï": {
+    "matchTags": ["amenity/fast_food"],
```
Comment (pyrog) [inline brands/amenity/restaurant.json:2710]: Could be either a fast-food, either a restaurant.  https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#nomatch
```suggestion
    "nomatch": ["amenity/fast_food|Pizza Paï"],
```

Comment (pyrog) [reply brands/amenity/restaurant.json:2710]: and a similar entry should be added in [brands/amenity/fast_food.json](https://github.com/osmlab/name-suggestion-index/blob/master/brands/amenity/fast_food.json)

**Comment by pyrog** [comment]
In https://github.com/osmlab/name-suggestion-index/pull/3681/commits/ff8764fdac06dded0a8151ec3e6a4e42ac746d20 changes don't rely to PR 😉 
Also, could you rebase, please ?

File: brands/amenity/fast_food.json [hunk 2]
```diff
@@ -3352,6 +3352,22 @@
...
   },
+  "amenity/fast_food|Pizza Paï": {
+    "nomatch": [
+      "amenity/restaurant|Pizza Paï"
+    ],
+    "countryCodes": [
+      "fr"
+    ],
+    "tags": {
+      "amenity": "restaurant",
```
Comment (pyrog) [inline brands/amenity/fast_food.json:3363]:
```suggestion
      "amenity": "fast_food",
```

**Comment by michalisKout** [comment]
Thanks for the information guys. I just add the Pizza Pai in fast_food.json.
@pyrog sorry for the last 2 commits, i made a rebase. AutoFormat on save was true in my editor...

**Comment by Adamant36** [comment]
Thanks. Looks fine.

**Comment by pyrog** [comment]
Thanks too 😃

**Pull request merged**